Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -632,16 +637,21 @@ - (void)performLogOut
          [self deauthorizeDevice];
        }
 
-       NSString *msg = [NSString stringWithFormat:@"Error signing out for barcode %@",
-                        currentBarcode.md5String];
-       [NYPLErrorLogger logNetworkError:error
-                                   code:NYPLErrorCodeApiCall
-                                summary:@"signOut"
-                                request:request
-                               response:response
-                                message:msg
-                               metadata:nil];
-       [self showLogoutAlertWithError:error responseCode:statusCode];
+       if (!self.businessLogic.selectedAuthentication.isOauth) {
```

Comment:
note: I realize now that this `if` is probably wrong. I added it to avoid the error message to show up for OAuth sign-outs thinking it would just apply to barcode sign-outs. But I think this whole sign-out logic is actually wrong: the sign-out request is identical to the sign-in request. So before making any more changes I filed SIMPLY-3208 to fix the sign-out call.

Reply:
for the mean time this `if` doesn't change much, so I'll leave it in.